config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -125,3 +125,98 @@ periodics:
         requests:
           cpu: 2
           memory: "6Gi"
+
+- name: ci-kubernetes-e2e-gci-gce-scalability-watch-list-off
+  cluster: k8s-infra-prow-build
+  interval: 6m
```

Comment:
Every 6 minutes? Sounds like an unnecessary often?

Reply:
ah, so this field controls frequency of the job :)
I misread it. Running the job once a day is find.
I will change it to 24h. Thanks!